cumulusci/tasks/push/tests/test_push_tasks.py

```diff
@@ -0,0 +1,481 @@
+        task()
+
+
+def test_report_push_status_error():
+    task = create_task(BaseSalesforcePushTask, options={})
+    task.sf = mock.MagicMock()
+    task.push_report = mock.MagicMock()
+    task.sf.query_all.return_value = {"totalSize": 1, "records": []}
+    with pytest.raises(PushApiObjectNotFound):
+        task._report_push_status("0DV1R000000k9dEWAQ")
+
+
+def test_get_push_request_job_results(
+    package_push_job_success, package_push_job_failure, package_push_job_cancel
+):
+    task = create_task(BaseSalesforcePushTask, options={})
+    task.sf = mock.MagicMock()
+    task.push_report = mock.MagicMock()
+    task.push_request = mock.MagicMock()
+    task.push_request.get_push_job_objs.return_value = [
+        package_push_job_success,
+        package_push_job_failure,
+        package_push_job_cancel,
+    ]
+    task._get_push_request_job_results()
+    task.push_request.get_push_job_objs.assert_called_once()
+
+
+def test_schedule_push_org_query_get_org_error():
+    task = create_task(
+        SchedulePushOrgQuery,
+        options={
+            "orgs": ORG,
+            "version": VERSION,
+            "namespace": NAMESPACE,
+            "start_time": None,
+            "batch_size": "200",
+            "min_version": "1.1",
+        },
+    )
+    task.push = mock.MagicMock()
+    task.push_api = mock.MagicMock()
+    task.sf = mock.Mock()
+    task.push_api.return_query_records = mock.MagicMock()
+    task.push_api.return_query_records.return_value = {"totalSize": "1"}
+    task.sf.query_all.return_value = PACKAGE_OBJS
+    with pytest.raises(ValueError):
+        assert task._get_orgs() == [NAME]
+
+
+def test_schedule_push_org_query_get_org():
+    task = create_task(
+        SchedulePushOrgQuery,
+        options={
+            "orgs": ORG,
+            "version": VERSION,
+            "namespace": NAMESPACE,
+            "start_time": None,
+            "batch_size": "200",
+            "subscriber_where": "OrgType = 'Sandbox'",
+        },
+    )
+    task.push = mock.MagicMock()
+    task.push_api = mock.MagicMock()
+    task.sf = mock.Mock()
+    task.sf.query_all.return_value = PACKAGE_OBJS
+    assert task._get_orgs() == ["bar", NAME]
+
+
+def test_schedule_push_org_list_run_task_with_time():
+    query = "SELECT Id, PackagePushRequestId, SubscriberOrganizationKey, Status FROM PackagePushJob WHERE Id = '0DV1R000000k9dEWAQ'"
+    with open(ORG_FILE, "w") as file:
+        file.write(f"{ORG_FILE_TEXT}")
+    task = create_task(
+        SchedulePushOrgList,
+        options={
+            "orgs": ORG_FILE,
+            "version": VERSION,
+            "start_time": "now",
+            "namespace": NAMESPACE,
+        },
+    )
+    task.push = mock.MagicMock()
+    task.sf = mock.MagicMock()
+    task.sf.query_all.return_value = PACKAGE_OBJS
+    task.push.create_push_request.return_value = (task.sf.query_all.return_value, 2)
+    task._run_task()
+    task.sf.query_all.assert_called_with(query)
```

Comment:
Currently this is only asserting that the task did _some_ query, which is a pretty vague assertion. I think the most important assertion to make is that task.push.create_push_request was called with the expected values.